rmgpy/data/kinetics/family.py

```diff
@@ -2085,8 +2085,7 @@ def retrieveOriginalEntry(self, templateLabel):
         
         Where the TrainingReactionEntry is only present if it comes from a training reaction
         """
-        
-        templateLabels = templateLabel.split()[0].split(';')
+        templateLabels = templateLabel.split()[-1].split(';')
```

Comment:
What improvement does this commit provide?  You don't seem to have changed the comment structure in this PR so if we need this commit to parse properly why does uncertainty parsing work currently?  Is this out of date?

Reply:
The method of parsing comments here relies on identifying the starting words of lines (the discussion and decision is in the conversation of this PR). Since the template label was at the start of the line, this method wasn't able to work. The uncertainty parsing involves a different scheme to identify line breaks (which is more verbose and has its own quirks), so it is not effected by this change.